DocumentDataAPI/DocumentDataAPI/Data/Repositories/NpgWordRelevanceRepository.cs

```diff
@@ -0,0 +1,39 @@
+using System.Data;
+using Dapper;
+using DocumentDataAPI.Data.Repositories.Helpers;
+
+namespace DocumentDataAPI.Data.Repositories
+{
+    public class NpgWordRelevanceRepository : IWordRelevanceRepository
+    {
+        private readonly IDbConnectionFactory _connectionFactory;
+        private readonly ILogger<NpgWordRelevanceRepository> _logger;
+        private readonly ISqlHelper _sqlHelper;
+
+        public NpgWordRelevanceRepository(IDbConnectionFactory connectionFactory, ILogger<NpgWordRelevanceRepository> logger,
+            ISqlHelper sqlHelper)
+        {
+            _connectionFactory = connectionFactory;
+            _logger = logger;
+            _sqlHelper = sqlHelper;
+        }
+
+        public async Task<int> UpdateWordRelevances()
+        {
+            _logger.LogDebug("Updating all TF-IDF scores in database");
+            const string script = @"update document_data.word_ratios w1
```

Comment:
```suggestion
            _logger.LogDebug("Updating all TF-IDF scores in database");
            // Cast a value to avoid integer division (default behavior in postgres)
            const string script = @"update document_data.word_ratios w1
```